internal/gengapic/options.go

```diff
@@ -149,3 +150,16 @@ func parseOptions(parameter *string) (*options, error) {
 
 	return &opts, nil
 }
+
+// Utility function for stringifying the Transport enum
+func (t transport) String() string {
```

Comment:
Nit: can we colocate methods with the type they belong to for ease of readability? Consider either moving the type def for `transport` down here or adding a new file `transport.go` (and a corresponding test file).